config/routes.rb

```diff
@@ -19,9 +19,12 @@
   end
 
   resources :news
+  resources :bug_tickets, except: [:update]
+  put 'bug_tickets/:id' => 'bug_tickets#update', defaults: { format: 'js' }
```

Comment:
yes. With this code the test for the update through Adjax passed. I google this, when I got the error.